test/kwokctl/kwokctl_binary.test.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DIR="$(dirname "${BASH_SOURCE[0]}")"
+
+DIR="$(realpath "${DIR}")"
+
+source "${DIR}/../helper.sh"
+source "${DIR}/helper.sh"
+
+function main() {
+  local all_releases=("${@}")
+  build_kwokctl
+  build_kwok
+
+  test_all "binary" "workable" "${all_releases[@]}" || exit 1
+}
+
+main $(cat "${DIR}"/../../supported_releases.txt)
```

Comment:
can we think of refactoring the `cat <PATH to >/supported_releases.txt`  code into helper or a common file, that way if we need to change the name we don't need to update it in every script.